app/controllers/api/v1/court_hearings_controller.rb

```diff
@@ -4,7 +4,7 @@ class CourtHearingsController < ApiController
       def create
         court_hearing = CourtHearing.create!(court_hearings_attributes)
 
-        if move
+        if should_save_court_hearings?
```

Comment:
`should_save_court_hearings` sounds like it should be `.save` in local,
what about
`should_save_in_nomis` or `should_be_saved_in_nomis`?

Reply:
I'll keep it consistent with the query param